radio/src/translations/de.h

```diff
@@ -1318,37 +1318,37 @@
 
 // The following content is Untranslated)
 
-#define TR_USE_THEME_COLOR             "Use theme color"
+#define TR_USE_THEME_COLOR             "Farbe des Themes verwenden"
 
-#define TR_ADD_ALL_TRIMS_TO_SUBTRIMS   "Add all Trims to Subtrims"
+#define TR_ADD_ALL_TRIMS_TO_SUBTRIMS   "Alle Trimmungen übernehmen"
 #if LCD_W > LCD_H
-  #define TR_OPEN_CHANNEL_MONITORS        "Open Channel Monitor"
+  #define TR_OPEN_CHANNEL_MONITORS        "Öffne Kanal Monitor"
 #else
-  #define TR_OPEN_CHANNEL_MONITORS        "Open Channel Mon."
+  #define TR_OPEN_CHANNEL_MONITORS        "Öffne Kanal Mon."
 #endif
-#define TR_DUPLICATE                   "Duplicate"
-#define TR_ACTIVATE                    "Set Active"
-#define TR_RED                         "Red"
-#define TR_BLUE                        "Blue"
-#define TR_GREEN                       "Green"
-#define TR_COLOR_PICKER                "Color Picker"
-#define TR_EDIT_THEME_DETAILS          "Edit Theme Details"
+#define TR_DUPLICATE                   "Duplizieren"
+#define TR_ACTIVATE                    "Aktivieren"
+#define TR_RED                         "Rot"
+#define TR_BLUE                        "Blau"
+#define TR_GREEN                       "Grün"
+#define TR_COLOR_PICKER                "Farbauswahl"
+#define TR_EDIT_THEME_DETAILS          "Theme Details Bearb."
 #define TR_AUTHOR                      "Author"
-#define TR_DESCRIPTION                 "Description"
-#define TR_SAVE                        "Save"
-#define TR_CANCEL                      "Cancel"
-#define TR_EDIT_THEME                  "EDIT THEME"
+#define TR_DESCRIPTION                 "Beschreibung"
+#define TR_SAVE                        "Speichern"
+#define TR_CANCEL                      "Abbruch"
+#define TR_EDIT_THEME                  "THEME Editieren"
 #define TR_DETAILS                     "Details"
 #define TR_THEME_EDITOR                "THEMES"
 
 // Main menu
-#define TR_MAIN_MENU_SELECT_MODEL      "Select\nModel"
-#define TR_MAIN_MENU_MODEL_NOTES       "Model\nNotes"
-#define TR_MAIN_MENU_CHANNEL_MONITOR   "Channel\nMonitor"
-#define TR_MAIN_MENU_MODEL_SETTINGS    "Model\nSettings"
-#define TR_MAIN_MENU_RADIO_SETTINGS    "Radio\nSettings"
-#define TR_MAIN_MENU_SCREEN_SETTINGS   "Screens\nSettings"
-#define TR_MAIN_MENU_RESET_TELEMETRY   "Reset\nTelemetry"
-#define TR_MAIN_MENU_STATISTICS        "Statistics"
-#define TR_MAIN_MENU_ABOUT_EDGETX      "About\nEdgeTX"
+#define TR_MAIN_MENU_SELECT_MODEL      "Wähle\nModell"
+#define TR_MAIN_MENU_MODEL_NOTES       "Model\nNotizen"
```

Comment:
"Modellnotizen"

Reply:
Ithink it's better with  \n  but one "l" is missing. done